java-symbolic-execution/src/test/files/se/Spring5Assert.java

```diff
@@ -0,0 +1,160 @@
+package symbolicexecution.behaviorcache;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.springframework.lang.Nullable;
+import org.springframework.util.Assert;
+
+class Spring5Assert {
+
+  Supplier<String> func = () -> "message";
```

Comment:
make it a constant (`private static final Supplier<String> FUNC = () -> "message";`), so it will be easier to recognize it later on in the code, due to the capitalized code.